src/pyuvdata/utils/ps_cat.py

```diff
@@ -0,0 +1,796 @@
+# -*- mode: python; coding: utf-8 -*-
```

Comment:
I don't like the name of this module -- I had no idea what it would mean before I looked at the code itself. I thought maybe "point source catalog" or "power spectrum catalog" or maybe something to do with concatenation...

It's probably better just to be like `phase_center_catalog`